TestCases/fea_topology/quick_start/settings_refnode.cfg

```diff
@@ -3,6 +3,9 @@ PHYSICAL_PROBLEM= ELASTICITY
 MATH_PROBLEM= DISCRETE_ADJOINT
 DYNAMIC_ANALYSIS= NO
 %
+SINGLEZONE_DRIVER=YES
+ITER= 1
```

Comment:
@rsanfer , @talbring can you guys please explain the current maximum iteration stopping criteria in an issue?

Reply:
With this we already went a little bit ahead of ourselves. We thought we would make progress a little bit faster. I am currently preparing a documentation for controlling the solver, where I will explain all of this. For the singlezone adjoint solver it is simply EXT_ITER -> ITER. Everything else is still the same.